src/metamodels/crosscat.py

```diff
@@ -228,11 +228,9 @@ class CrosscatMetamodel(metamodel.IBayesDBMetamodel):
     Internally, the Crosscat metamodel adds SQL tables to the database
     with names that begin with ``bayesdb_crosscat_``.
 
-    `crosscat` can be a crosscat engine instance or constructor. Prefer the
-    latter case, as that way bayeslite stochasticity is passed to the
-    constructor `seed` argument when engines are constructed for crosscat
-    queries (which at the time of writing happens every time bayeslite makes a
```

Comment:
When the provided constructor will be used seems like info a caller would want to have. Please keep that?